src/test/kotlin/org/rust/ide/inspections/import/AutoImportFixTest.kt

```diff
@@ -2228,6 +2228,34 @@ class AutoImportFixTest : AutoImportFixTestBase() {
         }
     """, checkOptimizeImports = false)
 
+    @MockEdition(CargoWorkspace.Edition.EDITION_2018)
+    fun `test insert import from different crate at correct location`() = checkAutoImportFixByFileTree("""
```

Comment:
This test doesn't actually check your changes - it passes even without them since `lib.rs` is a part of workspace

To makes it actually check your changes, you may want to use `dep-lib/lib.rs` instead of `lib.rs`. Don't forget to add `@ProjectDescriptor(WithDependencyRustProjectDescriptor::class)` annotation